receipt_dynamo/receipt_dynamo/entities/__init__.py

```
@@ -14,6 +14,10 @@
     CompactionLock,
     item_to_compaction_lock,
 )
+from receipt_dynamo.entities.compaction_run import (  # noqa: F401
```

Ruff (0.13.1)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)